stdlib/universe/holt_winters_test.go

```diff
@@ -19,6 +21,12 @@ import (
 	"github.com/influxdata/flux/stdlib/universe"
 )
 
+const epsilon float64 = 1e-03
```

Comment:
This seems really high. I wonder if the algorithm could be improved to be more stable... would it make sense to file an issue?

Reply:
Looking at Jonathan's [comment](https://influxdata.slack.com/archives/C0AD7FFS9/p1658242941737489?thread_ts=1658174400.602789&cid=C0AD7FFS9), it seems le-5 would work. So I will change it to that.

Reply:
It should be set to as small as we can and still have the test pass. `1e-05` still seems high to me to be honest, unless there is a huge tradeoff in terms of performance to get more stable answers.

Reply:
my mistake, what I meant was `1e-5`. we have epsilon [1e-6](https://github.com/influxdata/flux/blob/92b442939403cb3e159a42bc17f4e668d735af32/execute/executetest/transformation.go#L25) in the testing framework at the moment.

Reply:
Ah, the epsilon is already bigger than I thought it would be. I wonder why it's so high? In any case you are only moving it by one decimal place so that seems fine.